presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -2,6 +2,30 @@
 Deploying Presto
 ================
 
+Requirements
+------------
+
+Linux Operating System
```

Comment:
Using separate sections instead of nested lists to allow clean anchor usage and potentially more text